wikipedia-sentences/README.md

```diff
@@ -6,13 +6,15 @@
 
 ## Table of contents: 
 
+- [Overview](#overview)
+- [🐍 Build the app with Python](#-🐍-build-the-app-with-python)
```

Comment:
Remove the emoji from the `(#build-the-app-with-python)` I think that's the problem

Reply:
That's the situation [here](https://github.com/jina-ai/examples/tree/master/multires-lyrics-search) and it's not working.

Reply:
Are you manually creating toc? If so you can hover over the heading of the section and it should show a link icon on the left. Then you can right-click and copy link

Reply:
that shows the full url? 

`https://github.com/jina-ai/examples/blob/9d9de0bade9bb6498c266aab3f6bfdb4cba3ddf2/wikipedia-sentences/README.md#-build-the-app-with-python`

Reply:
I tried just adding the ending #-build-the-app-with-python but it didn't work